### DISCUSSION TITLE
Build instructions?

### DISCUSSION THREAD

**Discussion by mrghosti3** [GyulyVGC/sniffnet#7] [General]
Could there be added instructions on how to compile the project? This would be useful for people who might want to join in coding. For example it might be good to write what dependencies might be required to build (ex. for linux libpcap is required).

**Comment by GyulyVGC** [comment]
There is a section of the README (Error conditions) in which I specified Windows and Linux dependencies; on MacOS it has no dependencies.

Do you think it would be better to write those in a separate section? 

Also, if you notice some information is missing in that section let me know about that.

> Could there be added instructions on how to compile the project? This would be useful for people who might want to join in coding. For example it might be good to write what dependencies might be required to build (ex. for linux libpcap is required).

**Reply by mrghosti3** [reply]
Yeah, I think it would be best to have a seperate section for external lob depencies.

Other than that everything seems nicely in that section.

**Reply by GyulyVGC** [reply]
Added a subsection in “Installation” in the README, I hope it’s clearer now!

**Reply by mrghosti3** [reply]
Yeah. Thank you very much.
